### PR TITLE
oooo dev branch forgor this 💀 

### PR DESCRIPTION
**Pull request by lunaticwhat**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [x] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/BuildConfig.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+        package emu.grasscutter;
+        public class BuildConfig {
```
Comment (ghost) [inline src/main/java/emu/grasscutter/BuildConfig.java]: Remove.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/BuildConfig.java (hunk 1 above)
Comment (nitrog0d) [reply src/main/java/emu/grasscutter/BuildConfig.java]: toxic

Comment (nitrog0d) [reply src/main/java/emu/grasscutter/BuildConfig.java]: at least ask nicely like "Hi Lunatic please remove this line 🥺 uwu"

**Review by ghost — APPROVED** [review]

File: src/main/java/emu/grasscutter/BuildConfig.java (hunk 1 above)
Comment (lunaticwhat) [reply src/main/java/emu/grasscutter/BuildConfig.java]: > Remove.

actually you have to remove it because you are the @WriteAccess 💀

**Pull request merged**